src/components/availability-panel/drag-drop-context/DroppableComponent.tsx

```diff
@@ -1,25 +1,94 @@
-import { FC } from 'react';
-import { Droppable } from 'react-beautiful-dnd';
-import { droppableComponent } from '@/interfaces/availabilityPanel.type';
-import DraggableComponent from '@/components/availability-panel/drag-drop-context//DraggableComponent';
+import { FC } from "react";
+import { Droppable, Draggable } from "react-beautiful-dnd";
+import classNames from "@/components/availability-panel/drag-drop-context/styles.module.scss";
+import { droppableComponent } from "@/interfaces/availabilityPanel.type";
+import idleMembers from "@/pages/idle-members";
```

Comment:
Could you look into prettier settings please, I believe we are following single quote here